pages/linux/dash.md

```diff
@@ -0,0 +1,20 @@
+# dash
+
+> Debian Almquist Shell. Roughly 4x times faster than Bash.
```

Comment:
Let's get rid of the 2nd sentence. We would want the descriptions to be as objective as possible.